tests/test_odcs_util.py

```diff
@@ -111,14 +117,17 @@ def handle_composes_post(request):
         assert body_json['source'].get('sigkeys') == sigkeys
         assert body_json.get('flags') == flags
         assert body_json.get('arches') == arches
+        assert body_json.get('multilib_arches') == multilib_arches
+        assert body_json.get('multilib_method') == multilib_method or MULTILIB_METHOD_DEFAULT
```

Comment:
This is repeating the logic of the code under test. Would prefer to see an expected value in the parametrization.